File: rest/making-calls-sip/example-3/example-3.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# Initialize Twilio Client
@client = Twilio::REST::Client.new(account_sid, auth_token)

# Get the accounts with the given sid
@accounts = @client.api.v2010.accounts(account_sid)

@call = @accounts.calls.create(
  url: 'http://www.example.com/sipdial.xml',
  to: 'sip:kate@example.com?hatchkey=4815162342',
  from: 'Jack'
)

# Print the call direction
puts @call.direction

File: fax/sip-send/example-3/example-3.5.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once '/path/to/vendor/autoload.php'; // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "AC850685e1d9b8c09dae0b938923dc0d42";
$token = "your_auth_token";
$client = new Client($sid, $token);

$options = array(
  "from" => "Jack"
);
$fax = $client->fax->v1->faxes->create(
  "sip:kate@example.com?hatchkey=4815162342;transport=TCP",
  "https://www.twilio.com/docs/documents/25/justthefaxmaam.pdf",
  $options
);

echo $fax->sid;
